src/dynarec/la64/dynarec_la64_helper.c

```diff
@@ -820,6 +820,12 @@ int sse_setround(dynarec_la64_t* dyn, int ninst, int s1, int s2)
     return s2;
 }
 
+
+void x87_purgecache(dynarec_la64_t* dyn, int ninst, int next, int s1, int s2, int s3)
+{
+    // TODO
```

Comment:
:O don't that break games?

Reply:
well, there is no x87 support, so nothing to do for now, no?

Reply:
Ah yeah, fair point.